charts/scalardl-audit/files/envoy_grafana_dashboard.json

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
This is a copy from `charts/scalardl/files/envoy_grafana_dashboard.json` so no need to look at.